File: joseffe-barroso/Python/aula04/while/exercise44.py
quantity = 10
sumNumbers = 0
higherNumber = 0

index = 0
while (index < quantity):
  number = int(input("Digite um número: "))

  if (index == 0):
    higherNumber = number
  elif (number > higherNumber):
    higherNumber = number

  sumNumbers = sumNumbers + number
  index = index + 1

avarage = sumNumbers / quantity

print("média", avarage)
print("soma", sumNumbers)
print("maior", higherNumber)

File: joseffe-barroso/Java/src/entrada_processamento_saida/aula04/While/Exercise44.java
import java.util.Scanner;

public class Exercise44 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);

		int quantity = 0, sumNumbers = 0, higherNumber = 0, index, number, avarage;

		index = 0;
		while (index < quantity) {
			System.out.println("Qual é o valor da sequencia?");
			number = scanner.nextInt();

			if (index == 0) {
				higherNumber = number;
			} else if (number > higherNumber) {
				higherNumber = number;
			}

			sumNumbers = sumNumbers + number;
			index = index + 1;
			
			avarage = sumNumbers / quantity;
			
			System.out.printf("\nMédia %d", avarage);
			System.out.printf("\nSoma %d", sumNumbers);
			System.out.printf("\nMaior %d", higherNumber);
			
			scanner.close();
		}
	}

}
